src/main/java/graphql/nadel/hooks/HydrationArguments.java

```diff
@@ -0,0 +1,61 @@
+package graphql.nadel.hooks;
+
+import graphql.PublicApi;
+import graphql.language.Argument;
+import graphql.schema.GraphQLArgument;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+@PublicApi
+public class HydrationArguments {
```

Comment:
I could imagine we make this more convenient by providing get(name), getValue(name) like methods

Reply:
added these methods